challenge_gpsr/src/gpsr.py

```diff
@@ -19,6 +19,196 @@
 
 from robocup_knowledge import load_knowledge
 
+from conversation_engine import ConversationEngine
+
+class ConversationEngineWithHmi(ConversationEngine):
```

Comment:
@LoyVanBeek This should be moved to conversation_engine.

Reply:
@LoyVanBeek ping ;)

Reply:
This is the core of the challenge, I'd rather keep this right here.

Reply:
also not move it to a separate file in this package?